### DISCUSSION TITLE
how to update?

### DISCUSSION THREAD

**Discussion by ghost** [OpenRCT2/OpenRCT2#23102] [Q&A] [answered]
if a new openrct2 is there, how to update my old version so that i don't loose any savegames?

Do i need to uninstall the old version or just install a new one?

**Comment by telk5093** [comment]
You savegames are stored in ``(Documents)/OpenRCT2``, especially in the ``save`` folder.
It will not be losed unless you don't remove ``(Documents)/OpenRCT2`` folder.
I recommend to use [OpenLauncher](https://github.com/OpenRCT2/OpenLauncher/releases) to update OpenRCT2 easily

**Comment by ghost** [comment]
How you install openlauncher?

**Reply by telk5093** [reply, marked as answer]
> How you install openlauncher?

I've linked above already. Just download openlauncher.exe and install it.
![image](https://github.com/user-attachments/assets/7efb15ef-77f7-4e89-8266-60877590705f)

**Comment by ghost** [comment]
I can't click the play button

![afbeelding](https://github.com/user-attachments/assets/9b251157-070d-4a78-8de6-df5b058afbf9)

**Reply by Harry-Hopkinson** [reply]
You have to click "Download" first. Note that you can also select "Show pre-release versions" for the latest builds between Release versions.
